### PR TITLE
Change name from block to format

### PR DESCRIPTION
**Pull request by karmatosed**
Just a name change, needs investigation into what code needs to change also.

### CODE REVIEW

**Comment by mtias** [comment]
I also like not having the word "block" exposed to users.

File: editor/edit-post/sidebar/header.js [hunk 1]
```diff
@@ -32,9 +32,9 @@ const SidebarHeader = ( { panel, onSetPanel, onToggleSidebar, count } ) => {
 			<button
 				onClick={ () => onSetPanel( 'block' ) }
 				className={ `editor-sidebar__panel-tab ${ panel === 'block' ? 'is-active' : '' }` }
-				aria-label={ __( 'Block settings' ) }
+				aria-label={ __( 'Format settings' ) }
 			>
-				{ sprintf( _n( 'Block', '%d Blocks', count ), count ) }
+				{ __( 'Format' ) }
```
Comment (mtias) [inline editor/edit-post/sidebar/header.js]: We'd still want the count to be used, just not the Block(s) variation.

Comment (karmatosed) [reply editor/edit-post/sidebar/header.js]: I think I've done that now.

**Comment by carlhancock** [comment]
@mtias @karmatosed Why the desire to move away from referring to the block settings as simply Block in the sidebar? Format doesn't seem to make as much sense as it may have nothing to do with formatting.  The settings sidebar is going to be known as just that... the settings sidebar.  And that tab would house the settings for a specific block you are editing. Why would you not want to simply refer to it as "Block"? It seems to make a lot more sense than "Format".

**Comment by travislopes** [comment]
I think changing the sidebar panel name from "Block" to "Format" makes sense in the context of the blocks provided with Gutenberg. However, there are going to be plenty of blocks made where, in the sidebar panel, you won't be selecting from format related settings but rather settings related to the content populating the block.

If we're moving away from the "Block" sidebar panel name, it should be more customizable for instances where "Format" doesn't fit, even if that means there's a whitelist of options you can choose from during block registration rather than being able to define a custom label.

**Comment by amandablum** [comment]
The lexicon here is important as it has UI implications. While "block format" is probably a concise way of differentiating what the sidebar does vs the ellipses or inserter, unless it has the word "block" in it, it just serves to further confuse the end user.

**Comment by greatislander** [comment]
I would agree that this change has some strange implications, given that the inserter (and lots of other places in the UI) still use "Blocks" as the terminology.

### Example 1: "Insert Block"

insert block

### Example 2: "Search for a block"

search for a block

### Example 3: "Blocks / Formatting"

formatting blocks

This last one is most confusing; there is a whole group of blocks that are identified as "Formatting" blocks.

Conceptually, I'd interpreted the sidebar as two levels of settings: document (post)-level and block-level. And indeed, choosing "Settings" from a block's contextual menu opens this tab. In my opinion, the change to "Format" makes the scope of this panel less clear. Maybe group the two tabs, "Document" and "Block", under a sidebar heading that reads "Settings"?

**Comment by amandablum** [comment]
I think these conversations are still lacking a broader concept- one I brought up in #4305 which is these words have meaning, but we seem to need clarity on the UX of what one can do from the settings panel vs the inserter and the ellipsis. Once we have more clarity on the difference between the inserter, ellipses and settings, these word choices, the hierarchy of lexicon choices, etc will all become easier. 

Block settings are really block specific styles, by the time you arrive there, you have already chosen a format by choosing a block type itself. So block format is really block type, IMO. The settings panel allows you to affect Block Styles. And the ellipses: Block Options.  But since format, styles and options are all very generic words, their meaning to us will likely just be confusing to an end user. It raises the question of why we require so many different places to affect one item.

**Comment by jasmussen** [comment]
One of the reasons I personally like the change, is that it suggests the content of the panel, and doesn't require a casual user to know what "a block" is.

Notably, the content of the Format panel is largely that — formatting options for what is selected. One future enhancement that might happen, is that you can apply formatting across multiple blocks.

**Comment by greatislander** [comment]
@jasmussen My point was mainly that the interface uses “block” everywhere so by the time a user gets to the point where they are adjusting block settings they’ve already encountered the term in the inserted interface, etc.

**Comment by karmatosed** [comment]
Firstly, thanks for everyone's comments. It's good to get feedback and yes we can always reconsider any commit - we should not avoid discussion just because something has got in, nothing is final.

I think a few points around this are important to note:
- We know the word block but a user.. they won't. 
- Removing the word block from other places totally should be an option.
- There is an existing pattern of the word 'format' in applications such as 'keynote' : some (only some) end users would know this.

Now all the above doesn't mean we're tied to having 'format' as the be all here.

> Why the desire to move away from referring to the block settings as simply Block in the sidebar? Format doesn't seem to make as much sense as it may have nothing to do with formatting.

@carlhancock, in this case I do think 'format' of the element is what this interaction area does.

@travislopes also I think my point relates to your comment. It's a good boundary and definition for us all to work through. What exactly will go there and what do we mean. I still stand by that we need to remove as much as possible the word block.. it's meaningless to users.

> The lexicon here is important as it has UI implications. While "block format" is probably a concise way of differentiating what the sidebar does vs the ellipses or inserter, unless it has the word "block" in it, it just serves to further confuse the end user.

@amandablum right now 'block' is confusing users. So we have a bit of a problem to fix. To an end user they do not care and really shouldn't have to about the term block.

> I would agree that this change has some strange implications, given that the inserter (and lots of other places in the UI) still use "Blocks" as the terminology.


@greatislander valid points, looping back maybe we should also iterate those. I really would not myself mind at all if we removed the word block everywhere. That's a totally personal opinion though right now and something we as a group need to discuss.

> This last one is most confusing; there is a whole group of blocks that are identified as "Formatting" blocks.

Valid point again, something I think we could change and iterate on. What would be better to say here?

Let's continue the discussion as it's important to think about not just our headspace, think about users and those that don't know what (or even should know) a block is.

**Comment by carlhancock** [comment]
@karmatosed  You need to think beyond core and the blocks provided by core.  Plugins are going to introduce a wide variety of blocks and the settings sidebar is most definitely going to be used for settings.  

If we provide a block that has conditional logic as one of the features of the block what makes more sense to explain to the user "Go to the settings for the block and configure the conditional logic..." or "Go to the formats for the thing that isn't called a block and configure the conditional logic...".

Of course users don't know what a block is.  Gutenberg is brand new and not even in core yet.  It's something users would learn and learn quickly. Plugins are going to be shoving blocks down their throats at a rapid clip once Gutenberg is released in 5.0. 

Plugins and themes are going to refer to them as blocks.  Documentation, blog posts, etc. are going to refer to them as blocks.  When we explain to a user how to add our plugin to their page or post... we are going to provide instructions on how to add our block to the page or post.

If they aren't called blocks what would they be called? The implications go well beyond core itself.

**Comment by spencerfinnell** [comment]
There is also the existing usage of the term "Post Formats" which historically have changed the entire post (document) when adjusted instead of the selected block.

Also along with @carlhancock what do you call them? I couldn't write that last sentence without referencing the word block.

**Comment by amandablum** [comment]
> @amandablum right now 'block' is confusing users. So we have a bit of a problem to fix. To an end user they do not care and really shouldn't have to about the term block.

Block is a noun. Format can have both noun and verb implications and we use it that way throughout WordPress, which is why the word is  so loaded. 

@karmatosed  I fundamentally disagree with this notion. Blocks are the basis of Gutenberg. We need some way to describe them and "widgets" is taken (but an interesting point that its how widgets probably got that name and perhaps could be repurposed here).  If these blocks don't have a nomenclature, then how will people understand them as entities they can affect? Without block being a singular entity, how do they know the settings affect that block entity rather than the page, or all the blocks?  If people don't know what blocks are, we need WordPress to better explain that when they install Gutenberg. Or find another word for them.. but that word should probably be a noun.

**Comment by afercia** [comment]
> the content of the Format panel is largely that — formatting options for what is selected. 

This is a bit arguable, as that panel contains also blocks metadata, as also noted in the [design principles](https://wordpress.org/gutenberg/handbook/reference/design-principles/#advanced-settings). For example, I wouldn't say the followings are related to "format":
image alt text 
heading level
heading HTML anchor
gallery Link to

> Removing the word block from other places totally should be an option.

How to replace, for example, "Insert block"? Hm, "Insert ..."? Jokes apart, it's been 12 months the Gutenberg terminology has been based on the word "block", tons of posts and articles out there refer to the Gutenberg "blocks". I think users will be fine with the notion of "block".

**Comment by greatislander** [comment]
See, for example: https://humanmade.com/2017/12/20/gutenberg-a-white-paper-for-agencies-and-enterprise/

"Starting with Blocks: Reimagining WordPress with Gutenberg"

**Comment by karmatosed** [comment]
For now, reverting and we can discuss in #4415. It's important to note that changing from block is likely, it's what we change it to that needs discussion. As the thoughts here came after the commit, it's right to revert and then consider in an issue where everyone can bring those thoughts, as oppose to this closed PR.

**Pull request merged**